SetupTearDown.ps1

```diff
@@ -29,6 +41,122 @@ Function SetupRemoting{
 
 } # SetupRemoting()
 
+Function SetupSshRemotingOnHost{
+
+    Write-Host "Enabling SSH Remoting on host computer..."
+
+    if (-NOT (Test-Path "$env:USERPROFILE\.ssh\id_ed25519.pub"))
+    {
+        Write-Host "`nGenerating SSH Public Key"
+        ssh-keygen -t ed25519
+    }
+    else
+    {
+        Write-Host "`nUsing existing SSH Public Key"
+    }
+    $authorizedKey = Get-Content -Path $env:USERPROFILE\.ssh\id_ed25519.pub
+
+    Write-Host "`nConfigure SSH-Agent with Private Key"
+    Get-Service ssh-agent | Set-Service -StartupType Automatic
+    Start-Service ssh-agent
+    ssh-add $env:USERPROFILE\.ssh\id_ed25519
+
+    if (-NOT (Test-Path "$env:ProgramFiles\PowerShell\7\"))
+    {
+        Write-Host "`nInstall PowerShell"
+        Invoke-WebRequest https://github.com/PowerShell/PowerShell/releases/download/v7.4.0/PowerShell-7.4.0-win-x64.msi -OutFile "$env:Temp\PowerShell-7.4.0-win-x64.msi"
+        msiexec.exe /package "$env:Temp\PowerShell-7.4.0-win-x64.msi" /quiet ADD_EXPLORER_CONTEXT_MENU_OPENPOWERSHELL=1 ADD_FILE_CONTEXT_MENU_RUNPOWERSHELL=1 ENABLE_PSREMOTING=1 REGISTER_MANIFEST=1 USE_MU=1 ENABLE_MU=1 ADD_PATH=1
+    }
+    else
+    {
+        Write-Host "`nPowerShell 7 already installed"
+    }
+
+    Write-Host "`nDone"
+    
+    Write-Host "`n`nRun the following command in each of the containers"
+    Write-Host ".\SetUpTearDown.ps1 -SetupContainer -AuthorizedKey '$authorizedKey'"
+
+} # SetupSshRemotingOnHost()
+
+
+Function SetupSshRemotingOnContainer{
+param(
+    [Parameter(Mandatory=$True)] [string]$AuthorizedKey

```

Comment:
Can this be an empty string?  If not, then add basic parameter validation
 [ValidateScript({-Not [String]::IsNullOrWhiteSpace($_)})]

Reply:
I think making the parameter mandatory is already doing that. When I try to pass in empty string it gives me an error. Do you still want me to add the validation?

![image](https://github.com/microsoft/NetPerfTest/assets/157108493/757665ad-4625-431d-b0f4-c72abb34c221)

Reply:
What about a string with spaces.  We should validate

Reply:
Ok. Fixed it.